pkg/operator/ceph/cluster/rbd/mirror.go

```diff
@@ -37,8 +38,9 @@ const (
 	appName = "rook-ceph-rbd-mirror"
 )
 
-// Cluster represents the Rook and environment configuration settings needed to set up rbd mirroring.
-type Mirroring struct {
+// ClusterConfig represents the Rook and environment configuration settings needed to set up rbd mirroring.
+type ClusterConfig struct {
```

Comment:
Although this makes perfect sense from a consistency point of view, that's not the original goal of the PR and not mentioned in your commit message.